lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -140,6 +141,22 @@ def hub_release_exists(semver)
       false
     end
 
+    def hub_find_remote_tag(tag_name)
+      tries = 1
+      # Attempt to find the build. Rescue and re-attempt if the build can not
+      # be found on github yet.
+      begin
+        # Start with a sleep because the tag generally takes about a second
+        # to show up on github.
+        sleep tries * tries
```

Comment:
I agree with the exponential retry, but with the default retry value this can take up to 20 mins. Is that okay? I doubt it will take that long for github to realise there was a tag created.